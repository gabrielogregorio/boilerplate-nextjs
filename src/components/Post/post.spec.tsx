import React from 'react'
import renderer from 'react-test-renderer'
import { Post } from '.'

it('renders post', () => {
  const tree = renderer.create(<Post />)
  expect(tree).toMatchSnapshot()
})
