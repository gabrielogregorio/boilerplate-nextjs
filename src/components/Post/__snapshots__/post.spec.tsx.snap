// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders post 1`] = `
<div>
  Post 1
</div>
`;
